### PR TITLE
Add affiliate links

### PR DESCRIPTION
**Pull request by tonyroberts**

### CODE REVIEW

File: content/pages/index.md [hunk 1]
```diff
@@ -43,3 +43,13 @@ This package collects utilities that require both `xlrd` and `xlwt`, including t
...
 There is a [Google Group](http://groups.google.com/group/python-excel) dedicated to working with Excel files in Python, including the libraries listed above along with manipulating the Excel application via COM. 
+
+## Commercial Development
+
+The following companies can provide commercial software development and consultancy and are specialists in working with Excel files in Python:
+
+<div class="affiliate-links"></div>
+
+| [![Clark Consulting & Research]({static}/images/ccr_python_excel.png)](http://www.clark-consulting.eu/) | [LingInfo](http://www.lexicon.net/sjmachin){.linginfo} |
+| ------------------------------------------------------------------------------------------------------- | --- |
```
Comment (cjw296) [inline content/pages/index.md]: @sjmachin has retired, so no need for the lexicon.net link.

**Review by cjw296 — CHANGES_REQUESTED** [review]
What needs to happen for the Github Action to trigger on this branch?

File: content/pages/index.md (hunk 1 above)
Comment (tonyroberts) [reply content/pages/index.md]: ok, I'll take that off then

**Comment by cjw296** [comment]
https://help.github.com/en/actions/reference/events-that-trigger-workflows#example-using-a-list-of-events

Guessing there's an "keep some stuff as an Artifact" around for PRs too.

File: content/pages/index.md (hunk 1 above)
Comment (tonyroberts) [reply content/pages/index.md]: done!

**Comment by cjw296** [comment]
rebase of this shoud well do it?

**Comment by cjw296** [comment]
No link to your own or @fzumstein 's companies?

**Comment by tonyroberts** [comment]
I figured it would make sense to get it close enough to what's currently there as a baseline before making any material changes.

**Comment by tonyroberts** [comment]
> rebase of this shoud well do it?

sure, or you can squash and merge it

**Pull request merged**